test/helpers/kubectl.go

```diff
@@ -444,7 +452,10 @@ func (kub *Kubectl) WaitforPods(namespace string, filter string, timeout time.Du
 		}).Info("WaitforPods: pods are not ready")
 		return false
 	}
-	return WithTimeout(body, "could not get Pods", &TimeoutConfig{Timeout: timeout})
+	return WithTimeout(
+		body,
+		fmt.Sprintf("could not get pods on filter %s", filter),
```

Comment:
error is a bit cryptic, what about "timed out waiting for pods with filter %s to be ready"?